pkg/get/tools.go

```diff
@@ -2967,5 +2967,33 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 				`,
 		})
 
+	tools = append(tools,
+		Tool{
+			Owner:       "sachaos",
+			Repo:        "viddy",
+			Name:        "viddy",
+			Description: "Validate your Kubernetes configuration files, supports multiple Kubernetes versions",
```

Comment:
Where is the description from?

It should be:

A modern watch command. Time machine and pager etc.


https://github.com/sachaos/viddy